_posts/2022-01-30-my-journey-in-openfaas-community.md

```diff
@@ -0,0 +1,64 @@
+---
+title: My Journey in OpenFaaS Community So Far
```

Comment:
How about? "My Journey Contributing To The OpenFaaS Community So Far"

Reply:
Thank You for suggestion. Changes are done.